### ISSUE TITLE
Update package_info_plus dependency

### ISSUE THREAD

**Issue by michalcickan** [reown-com/reown_flutter#9]
**Is your feature request related to a problem? Please describe.**
I have the newest package_info_plus version, and reown_core still depends on the older version, 7.0.0. Now the error looks like this:

```
Because no versions of reown_walletkit match >1.0.1 <2.0.0 and reown_walletkit 1.0.1 depends on reown_core ^1.0.0, reown_walletkit ^1.0.1 requires reown_core ^1.0.0.
Because reown_core 1.0.0 depends on package_info_plus ^7.0.0 and no versions of reown_core match >1.0.0 <2.0.0, reown_core ^1.0.0 requires package_info_plus ^7.0.0.
Thus, reown_walletkit ^1.0.1 requires package_info_plus ^7.0.0.
```

**Describe the solution you'd like**
Please update dependency to ^8.0.0

**Describe alternatives you've considered**
Now, I use dependency override, what is not a great solution.

```
dependency_overrides:
  package_info_plus: ^8.0.2
```

**Comment by quetool** [comment]
Hello @michalcickan! Thanks for opening, using dependency override it's the right solution right now IMHO. I'll be upgrading all dependencies ASAP anyway.

**Comment by quetool** [comment]
@michalcickan ! major decency upgrades are going to be held during our tech debt week, which should happen before EOY

**Comment by ahmed-tarek-salem** [comment]
Hi @michalcickan 

For me, I updated the package_info_plus to the latest version (8.1.0).
And it works fine.

**Comment by AlexV525** [comment]
@quetool Most of the fields are the same from v5 to v8. You can write `>=5.0.0 <9.0.0` to allow these versions.

**Comment by quetool** [comment]
Hello @michalcickan @ahmed-tarek-salem @AlexV525
This should be already fixed in `reown_walletkit: ^1.0.2`, could you try it out?
